stdlib/contrib/bonitoo-io/hex/hex.flux

```diff
@@ -0,0 +1,10 @@
+package hex
+
+builtin int : (v: string) => int
+builtin string : (v: A) => string
+builtin uint : (v: string) => uint
+builtin bytes : (v: string) => bytes
+
+toString = (tables=<-) => tables |> map(fn: (r) => ({r with _value: string(v: r._value)}))
```

Comment:
I'd like to remove these functions. They might be useful but I'd prefer not to add them until we see this pattern as something that is necessary.

Reply:
Since the flux data originate in InfluxDB per https://github.com/influxdata/feature-requests/issues/297, I supposed that it is a good idea to have short-hand functions that work over a table OOTB, I was mimicking the pattern of the built-in functions. They are now removed per your request.